src/WorkflowManager/appsettings.Development.json

```diff
@@ -26,10 +26,10 @@
     "storage": {
       "serviceAssemblyName": "Monai.Deploy.Storage.MinIO.MinIoStorageService, Monai.Deploy.Storage.MinIO",
       "settings": {
-        "endpoint": "localhost:9000",
```

Comment:
Can these not be changed, at least for appsettings.development as this is what is configured with the included docker-compose in the repo

Reply:
Fixed